docs/abi.rst

```diff
@@ -306,7 +321,7 @@ The supported methods are:
 
 * :any:`abi.StaticArray.__getitem__(index: int | Expr) <abi.StaticArray.__getitem__>`, used for :any:`abi.StaticArray` and :any:`abi.Address`
 * :any:`abi.Array.__getitem__(index: int | Expr) <abi.Array.__getitem__>`, used for :any:`abi.DynamicArray` and :any:`abi.String`
-* :any:`abi.Tuple.__getitem__(index: int) <abi.Tuple.__getitem__>`
+* :any:`abi.Tuple.__getitem__(index: int) <abi.Tuple.__getitem__>`, used for :any:`abi.NamedTuple`
 
 .. note::
```

Comment:
It would be nice to see a second note section here which explains that `NamedTuple` fields can be accessed by field name in addition to using `__getitem__`

Reply:
I somewhat re-ordered the following note, let me know how this should be rephrased.